test/studies/shootout/binary-trees/binarytrees-pool.chpl

```diff
@@ -8,31 +8,31 @@
 
 use Allocators;
 
-config const n = 10,         // the maximum tree depth
-             globalPoolSize = (2 ** 31)-1,
-             localPoolSize = globalPoolSize;
-
-var globalPool = new bumpPtrMemPool(globalPoolSize, alignment=0);
+config const n = 10;                         // the maximum tree depth
 
 proc main() {
-  const minDepth = 4,                      // the shallowest tree
-        maxDepth = max(minDepth + 2, n),   // the deepest normal tree
-        strDepth = maxDepth + 1,           // the depth of the "stretch" tree
-        depths = minDepth..maxDepth by 2;  // the range of depths to create
-  var stats: [depths] (int,int);           // stores statistics for the trees
+  const minDepth = 4,                        // the shallowest tree
+        maxDepth = max(minDepth + 2, n),     // the deepest normal tree
+        strDepth = maxDepth + 1,             // the depth of the "stretch" tree
+        depths = minDepth..maxDepth by 2,    // the range of depths to create
+        nodeSize = 24,                       // the approximate size of a node
+        poolSize = 2**(maxDepth+2)*nodeSize; // how much each pool allocates
+  var stats: [depths] (int,int);             // stores statistics for the trees
 
   //
   // Create the short-lived "stretch" tree, checksum it, and print its stats.
   //
   {
-    const strTree = newWithAllocator(globalPool, unmanaged Tree, strDepth, globalPool);
+    var pool = new bumpPtrMemPool(poolSize, alignment=0);
```

Comment:
I didn't catch this yesterday, but since bumpPtrMemPool is a class, this could be `const` since the pointer to the class isn't changing (it's fine that the class fields themselves will)—and that would permit both these declarations to share a `const` keyword which gzip will surely love (?).

[and ditto for the subsequent pools]